cranelift/codegen/src/machinst/lower.rs

```diff
@@ -358,9 +344,15 @@ impl<'func, I: VCodeInst> Lower<'func, I> {
         block_order: BlockLoweringOrder,
     ) -> CodegenResult<Lower<'func, I>> {
         let constants = VCodeConstants::with_capacity(f.dfg.constants.len());
-        let mut vcode = VCodeBuilder::new(abi, emit_info, block_order, constants);
+        let mut vcode = VCodeBuilder::new(
+            abi,
+            emit_info,
+            block_order,
+            constants,
+            /* backward = */ true,
```

Comment:
Done! The `Forward` option is actually dead, which totally makes sense, because we... don't have any forward VCode-to-VCode passes yet. I expect we will eventually, but IMHO it makes more sense to omit the enum arm (with a TODO explaining why) so that we can add it back only when we adequately test it.

(I'd just like to note as an aside that the dead-code analysis working to find unconstructed enum variants is *excellent*.)